tools/cronjobs_osgeo_lxd/compile_addons_git.sh

```diff
@@ -125,7 +125,7 @@ for c in "db" "display" "general" "gui/wxpython" "imagery" "misc" "raster" "rast
         MANBASEDIR="$path/docs/man" \
         SCRIPTDIR="$path/scripts" \
         ETC="$path/etc" \
-            SOURCE_URL="https://github.com/OSGeo/grass-addons/tree/master/grass${GRASS_VERSION}/" > \
+            SOURCE_URL="https://github.com/OSGeo/grass-addons/tree/grass${GRASS_VERSION}/src/" > \
```

Comment:
I would change (in the entire script) `GRASS_VERSION` -> `GMAJOR`

Reply:
I think this PR #613 solve it.